sites/partners/src/lib/listings/UnitsFormatter.ts

```diff
@@ -25,6 +25,10 @@ export default class UnitsFormatter extends Formatter {
         case "oneBdrm":
           unit.numBedrooms = 1
           break
+        case "studio":
+        case "SRO":
+          unit.numBedrooms = 0
+          break
```

Comment:
question(non-blocking): We don't have a use case for null number of bedrooms, so I wonder if we could make the default `0` instead. But that might be beyond this ticket

Reply:
I'd be fine to make that change, just waiting for an answer from product in the original support thread. Will get that answered and this merged in Monday's sync if I don't hear back!

Reply:
Got confirmation from product so updated that default!